tests/fixtures/mod.rs

```diff
@@ -62,41 +64,58 @@ fn check_auth_credentials(auth_header: Option<&Header>) -> bool {
 }
 
 pub fn get_file_count_with_pattern(pattern: &str, dir: &str) -> Result<usize, ()> {
-    // Command being run: `ls | grep .mp3 | wc -w`
-    let mut du_output_child = Command::new("ls")
-        .args(&[dir])
-        .stdout(Stdio::piped())
-        .spawn()
-        .unwrap();
-
-    if let Some(du_output) = du_output_child.stdout.take() {
-        let mut sort_output_child = Command::new("egrep")
-            .arg(pattern)
-            .stdin(du_output)
-            .stdout(Stdio::piped())
-            .spawn()
-            .unwrap();
-
-        du_output_child.wait().unwrap();
+    // Command being run: `ls | grep pattern | wc -w`
 
-        if let Some(sort_output) = sort_output_child.stdout.take() {
-            let head_output_child = Command::new("wc")
-                .args(&["-w"])
-                .stdin(sort_output)
-                .stdout(Stdio::piped())
-                .spawn()
-                .unwrap();
+    let cmd = {
+        Exec::shell(format!("ls {}", dir))
+            | Exec::shell(format!("grep '{}'", pattern))
+            | Exec::shell("wc -l")
```

Comment:
Maybe add a comment to indicate that it's an actual shell pipe

Reply:
Ok